project-jee8-webapp/src/main/java/fr/pantheonsorbonne/ufr27/miage/service/InfoCentreService.java

```diff
@@ -11,4 +11,6 @@
 
 	void periodicBulletin(List<Train> listTrains);
 
+	void periodicBulletin();
```

Comment:
this is for sending all the info we have on the train every 10s. That's what I understood you want to do, but I may be wrong.